index.html

```diff
@@ -304,21 +304,169 @@ <h5 class="work__Title--text">Multi-Post Stories Gain&plus;Glory</h5>
+                        <li class="frames__done__list"><a href="#" class="frames__done__link">RSpec</a></li>
+                    </ul>
+                    <ul class="frames__done__group2">
+                        <li class="frames__done__list"><a href="#" class="frames__done__link">Capybara</a></li>
+                        <li class="frames__done__list"><a href="#" class="frames__done__link">Selenium</a></li>
+                    </ul>
+                </div>
+            </div>
+
+
+            <div class="frame__content">
+                <div class="frame__content__Icon">
+                    <a href="#" class="frame__content__links">
+                        <img src="/img/icon-skills.svg" alt="langauge Icon" class="frame__content__img">
+                    </a>
+                </div>
+                <div class="frame__Title">
+                    <h5 class="frame__Title--text">Languages</h5>
+    
+                </div>
+                
+
+                <div class="frames__done">
+                    <ul class="frames__done__group">
+                        <li class="frames__done__list"><a href="#" class="frames__done__link">Codekit</a></li>
+                        <li class="frames__done__list"><a href="#" class="frames__done__link">GitHub</a></li>
+                        <li class="frames__done__list"><a href="#" class="frames__done__link">Codepen</a></li>
+                        <li class="frames__done__list"><a href="#" class="frames__done__link">Gitlab</a></li>
+                    </ul>
+                    <ul class="frames__done__group2">
+                        <li class="frames__done__list"><a href="#" class="frames__done__link">Terminal</a></li>
+                    </ul>
+                </div>
+            </div>
+
+        </section>
+
+        <footer>
+            <div class="footer__container">
+                <div class="FormText">
+                    <p class="FormText__para">
+                        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                    </p>
+                </div>
+    
+    
+                <div class="contact__container">
+    
+                    <div class="contact-us">
+                        <form action="#"name="simple-contact-form" accept-charset="utf-8" method="post">
+                            <label for="FullName"></label>
+                            <input id="FullName" name="Full Name" required="" type="text" placeholder="Full name" />
+                            <label for="contactEmail"></label>
+                            <input id="contactEmail" name="Email" required="" type="email" placeholder="Email adress"/>
+                            <label for="TextArea"></label>
+                            <textarea id="TextID" name="TextArea" minlength="15" maxlength="300"  required rows="7" placeholder="Enter Text"></textarea>
```

Comment:
Great job @solomonetchu 🤝

- [ ] Kindy make sure the `maxlenght="500"` property of the text area is as stated in the project requirements.